### DISCUSSION TITLE
Cant add multiple account on github web

### DISCUSSION THREAD

**Discussion by Sisableng** [community/community#81190] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, i dont know if is bug or not but i think its a little problem, i want to add multiple account on github website but after insert of authentications its going to 404 page [https://github.com/sessions/verified-device/mobile?auto=true]

![Screenshot (1)](https://github.com/community/community/assets/58748827/f8bb6dff-8ab4-4396-8fdd-e29d99acd0fd)

(https://github.com/sessions/verified-device/mobile?auto=true)

**Comment by yioannides** [comment]
I'm experiencing the same issue with the same url. I'm logged in on my smartphone, but I cannot log my second account on desktop.

**Reply by Superomarking** [reply]
Signing off the logged in account seems to be the fix.
